FunctionalEquivalence/FunctionalEquivalence.wdl

```diff
@@ -444,7 +442,7 @@ EOF
     >>>
 
     runtime {
-        docker: "us.gcr.io/broad-dsde-methods/functionalequivalence/fe_evaluation:1.0.0"
+        docker: "us.gcr.io/broad-dsde-methods/mgatzen/python-data-slim:98f9877658fbb4a8298fbd69f7daf0fa5415c6f79d7f4d0ef47a4ed41a5c5ad3"
```

Comment:
Why does this have a weird hash-looking version number? Could we use a more human-readable version, especially given you've bumped the dependency versions? It'd be nice if this version number was loosely correlated with those.